### PR TITLE
replaces all instances of pacid with facid

### PR DESCRIPTION
**Pull request by pudl**

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -98,11 +98,11 @@
 	icon_state = "bottle17"
 	spawned_reagent = "diethylamine"
 
-/obj/item/weapon/reagent_containers/glass/bottle/pacid
+/obj/item/weapon/reagent_containers/glass/bottle/facid
 	name = "Polytrinic Acid Bottle"
 	desc = "A small bottle. Contains a small amount of Polytrinic Acid"
```
Comment (hornygranny) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: no it aint

File: code/game/machinery/rechargestation.dm [hunk 2]
```diff
@@ -189,6 +189,6 @@
 				if(istype(R.module.emag, /obj/item/weapon/reagent_containers/spray))
 					var/obj/item/weapon/reagent_containers/spray/S = R.module.emag
 					if(S.name == "polyacid spray")
```
Comment (hornygranny) [inline code/game/machinery/rechargestation.dm]: no it aint

File: code/modules/reagents/reagent_containers/spray.dm [hunk 3]
```diff
@@ -52,7 +52,7 @@
 	if(reagents.has_reagent("sacid"))
 		message_admins("[key_name_admin(user)] fired sulphuric acid from \a [src].")
 		log_game("[key_name(user)] fired sulphuric acid from \a [src].")
-	if(reagents.has_reagent("pacid"))
+	if(reagents.has_reagent("facid"))
 		message_admins("[key_name_admin(user)] fired Polyacid from \a [src].")
 		log_game("[key_name(user)] fired Polyacid from \a [src].")
```
Comment (hornygranny) [inline code/modules/reagents/reagent_containers/spray.dm]: no it aint

File: code/modules/reagents/reagent_containers/spray.dm [hunk 4]
```diff
@@ -52,7 +52,7 @@
 	if(reagents.has_reagent("sacid"))
 		message_admins("[key_name_admin(user)] fired sulphuric acid from \a [src].")
 		log_game("[key_name(user)] fired sulphuric acid from \a [src].")
-	if(reagents.has_reagent("pacid"))
+	if(reagents.has_reagent("facid"))
 		message_admins("[key_name_admin(user)] fired Polyacid from \a [src].")
```
Comment (hornygranny) [inline code/modules/reagents/reagent_containers/spray.dm]: no it aint

**Comment by Anonus** [comment]
Doesn't fix #7273

**Comment by pudl** [comment]
jordie's fault, not mine

**Comment by Jordie0608** [comment]
#7128 Isn't merged and #6889 didn't touch sprays.

**Comment by pudl** [comment]
merge when

**Pull request merged**